src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -184,9 +217,6 @@ public ResponseEntity<Set<FaqDTO>> getFaqForCourse(@PathVariable Long courseId)
     @EnforceAtLeastStudent
```

Comment:
Since you already removed the other verification:
```suggestion
    @EnforceAtLeastStudentInCourse
```